src/tools/quemap/main.c

```diff
@@ -19,10 +19,10 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
  */
 
-#include <SDL2/SDL.h>
-
 #include "quemap.h"
 
+#include <SDL2/SDL.h>
+
```

Comment:
Just curious, we can't import SDL.h first? Typically I include external deps first, then our includes.